node.go

```diff
@@ -20,6 +20,7 @@ import (
 
 const (
 	statAverageExecTime = "avg_exec_time_ns"
+	statErrorCount      = "error_count"
```

Comment:
I think `errors` is a better name for the metric but I don't have a strong opinion.